src/main/resources/application.properties

```
@@ -1,6 +1,7 @@
 # OpenAPI/Swagger Configuration
 quarkus.smallrye-openapi.path=/openapi
 quarkus.swagger-ui.enable=true
+quarkus.swagger-ui.always-include=${ENABLE_SWAGGER:false}
```

why? zet het op true pls
```suggestion
quarkus.swagger-ui.always-include=${ENABLE_SWAGGER:true}
```